### PR TITLE
Hotfix/low stats[port to 1.11]

### PR DESCRIPTION
**Pull request by Raycoms**
# Changes proposed in this pull request:
- Equal to original pr

Review please

### CODE REVIEW

File: src/main/java/com/minecolonies/coremod/colony/ColonyView.java [hunk 1]
```diff
@@ -59,7 +59,7 @@
      */
     private Set<Block> freeBlocks = new HashSet<>();
 
-    private double overallHappiness;
+    private double overallHappiness = 5;
```
Comment (MinecoloniesSonar) [inline src/main/java/com/minecolonies/coremod/colony/ColonyView.java:62]: MAJOR Assign this magic number 5 to a well-named constant, and use the constant instead. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)

File: src/main/java/com/minecolonies/coremod/util/RecipeHandler.java [hunk 2]
```diff
@@ -75,7 +75,7 @@ public static void init(final boolean enableInDevelopmentFeatures, final boolean
         GameRegistry.addRecipe(new ShapedOreRecipe(new ItemStack(ModBlocks.blockHutWareHouse, 1), "XXX", "X#X", "XXX", 'X', PLANK_WOOD, '#', "chest")); // check
         GameRegistry.addRecipe(new ShapedOreRecipe(new ItemStack(ModBlocks.blockHutDeliveryman, 1), "XXX", "X#X", "XXX", 'X', PLANK_WOOD, '#', Items.LEATHER_BOOTS));
 
-        enableInDevelopmentFeatures(enableInDevelopmentFeatures);
+        //enableInDevelopmentFeatures(enableInDevelopmentFeatures);
```
Comment (MinecoloniesSonar) [inline src/main/java/com/minecolonies/coremod/util/RecipeHandler.java:78]: MINOR This block of commented-out lines of code should be removed. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)

**Comment by MinecoloniesSonar** [comment]
SonarQube analysis reported 45 issues
* ![MAJOR][MAJOR] 5 major
* ![MINOR][MINOR] 40 minor

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [ColonyView.java#L27](https://github.com/Minecolonies/minecolonies/blob/60a53b59b70bd372aef44c2058b90e666d9afa41/src/main/java/com/minecolonies/coremod/colony/ColonyView.java#L27): class "ColonyView" has 42 methods, which is greater than the 35 authorized. Split it into smaller classes. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1448)
1. ![MAJOR][MAJOR] [ColonyView.java#L93](https://github.com/Minecolonies/minecolonies/blob/60a53b59b70bd372aef44c2058b90e666d9afa41/src/main/java/com/minecolonies/coremod/colony/ColonyView.java#L93): Remove this unused method parameter "isNewSubScription". [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1172)
1. ![MAJOR][MAJOR] [ColonyView.java#L354](https://github.com/Minecolonies/minecolonies/blob/60a53b59b70bd372aef44c2058b90e666d9afa41/src/main/java/com/minecolonies/coremod/colony/ColonyView.java#L354): Provide multiple methods instead of using "isNewSubscription" to determine which action to take. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2301)
1. ![MAJOR][MAJOR] [RecipeHandler.java#L82](https://github.com/Minecolonies/minecolonies/blob/60a53b59b70bd372aef44c2058b90e666d9afa41/src/main/java/com/minecolonies/coremod/util/RecipeHandler.java#L82): Remove this unused private "enableInDevelopmentFeatures" method. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)
1. ![MINOR][MINOR] [blockout](https://github.com/Minecolonies/minecolonies/blob/60a53b59b70bd372aef44c2058b90e666d9afa41/src/main/java/com/minecolonies/blockout): Add a 'package-info.java' file to document the 'blockout' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [controls](https://github.com/Minecolonies/minecolonies/blob/60a53b59b70bd372aef44c2058b90e666d9afa41/src/main/java/com/minecolonies/blockout/controls): Add a 'package-info.java' file to document the 'controls' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [views](https://github.com/Minecolonies/minecolonies/blob/60a53b59b70bd372aef44c2058b90e666d9afa41/src/main/java/com/minecolonies/blockout/views): Add a 'package-info.java' file to document the 'views' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [blocks](https://github.com/Minecolonies/minecolonies/blob/60a53b59b70bd372aef44c2058b90e666d9afa41/src/main/java/com/minecolonies/coremod/blocks): Add a 'package-info.java' file to document the 'blocks' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [gui](https://github.com/Minecolonies/minecolonies/blob/60a53b59b70bd372aef44c2058b90e666d9afa41/src/main/java/com/minecolonies/coremod/client/gui): Add a 'package-info.java' file to document the 'gui' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [render](https://github.com/Minecolonies/minecolonies/blob/60a53b59b70bd372aef44c2058b90e666d9afa41/src/main/java/com/minecolonies/coremod/client/render): Add a 'package-info.java' file to document the 'render' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by marvin-bitterlich** [comment]
:shipit:

[Approved with PullApprove](https://pullapprove.com/Minecolonies/minecolonies/pull-request/977/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=Minecolonies/minecolonies)

**Pull request merged**